docs/cardinal/game/configuration.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: 'Configuration'
+description: 'Configuring the Cardinal game shard'
+---
+
+Each service in the World Engine stack reads its configuration values from environment variables. Using the [World CLI](https://github.com/Argus-Labs/world-cli) makes configuring your project even easier via its `world.toml` file, where all World Engine configuration values are in one place.
+
+Below are the key value pairs that can be set in either the `world.toml` file or directly in environment variables.
+
+
+## Cardinal `[cardinal]`
+| Name                         | Description                                                                                                                                                                                                                          | Required in Rollup Mode |
+|------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------|
+| CARDINAL_NAMESPACE           | The name of your Cardinal game shard. If you are running multiple game shards in a single World Engine instance, this MUST be unique for each game shard. Must only be alphanumeric characters and hyphens (-).                      | YES                     |
+| ROLLUP_MODE                  | When false, Cardinal will behave like a traditional game server. When true, Cardinal will use the EVM base shard to sequence game transactions, as well as enable the Router.                                                        | N/A                     |
```

Comment:
This should be "CARDINAL_ROLLUP_ENABLED". The description can remain the same.

https://github.com/Argus-Labs/world-engine/blob/main/cardinal/config.go#L53

Reply:
thanks, fixed.